app/views/users/edit.html.erb

```diff
@@ -15,8 +15,8 @@
 <table class="table table-striped table-bordered table-hover">
   <caption><span class="sr-only">List of guiders</span></caption>
   <colgroup>
-    <col width="98%">
-    <col width="1%">
+    <col width="91%">
+    <col width="9%">
```

Comment:
If the screenshot is up to date, it might be worth giving the 'actions' column a bit more room so the edit & delete buttons can sit on the same line?